scripts/update-sig-tables.py

```diff
@@ -65,6 +65,11 @@ def format_chat(chat):
             if chat.get('name') and chat.get('type')]
         )
 
+        short_name = None
+        for chat in sig.get('chat', []):
+            if chat.get('type') == 'slack':
+                short_name = chat.get('name').replace('#otel-', '')
```

Comment:
```suggestion
                short_name = chat.get('name').replace('#', '')
```

I would keep a kind of prefix for this, otherwise if we ever have a section called "Maintainers" or "Collector", etc we create a conflict. My preference would be "sig" but since we have some of them already maybe stick with otel?

Or we "urlify" the name?

Reply:
> My preference would be "sig"

makes sense to me, I added the `sig-` prefix (where it wasn't already prefixed by `sig-`